MAINTAINERS.yml

```diff
@@ -0,0 +1,1009 @@
+        - scripts/valgrind.supp
+        - soc/posix/
+        - subsys/debug/asan.c
+        - tests/boards/native_posix/
+    labels:
+        - "area: native port"
+    description: >-
+        POSIX architecture and SOC, native_posix board, and related drivers
+
+Networking:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - jukkar
+    collaborators:
+        - tbursztyka
+        - pfalcon
+    files:
+        - drivers/net/
+        - include/net/
+        - samples/net/
+        - subsys/net/
+    files-exclude:
+        - samples/net/sockets/coap_*/
+        - samples/net/lwm2m_client/
+        - subsys/net/lib/coap/
+        - subsys/net/lib/lwm2m/
+        - subsys/net/lib/openthread/
+        - subsys/net/lib/tls_credentials/
+    labels:
+        - "area: Networking"
+
+"Networking: BSD sockets":
+    # Provisional entry, to be corrected
+    maintainers:
+        - pfalcon
+    files:
+        - samples/net/sockets/
+        - subsys/net/lib/sockets/
+        - tests/net/socket/
+    labels:
+        - "area: Sockets"
+
+"Networking: Buffers":
+    # Provisional entry, to be corrected
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jhedberg
+    files:
+        - include/net/buf.h
+        - subsys/net/buf.c
+        - tests/net/buf/
+
+"Networking: CoAP":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - rveerama1
+    files:
+        - subsys/net/lib/coap/
+        - samples/net/sockets/coap_*/
+        - tests/net/lib/coap/
+
+"Networking: LWM2M":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mike-scott
+    files:
+        - samples/net/lwm2m_client/
+        - subsys/net/lib/lwm2m/
+    labels:
+        - "area: LWM2M"
+
+"Networking: MQTT":
+    # Provisional entry, to be corrected
+    maintainers:
+        - rlubos
+    collaborators:
+        - jukkar
+    files:
+        - subsys/net/lib/mqtt/
+        - tests/net/lib/mqtt_packet/
+        - samples/net/mqtt_publisher/
+
+"NIOS-2 arch":
+    # Provisional entry, to be corrected
+    # TODO: The entries in the maintainers spreadsheet do not match CODEWONERS at all!
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/nios2/
+        - include/arch/nios2/
+    labels:
+        - "area: NIOS2"
+
+"nRF52 BSIM":
+    status: maintained
+    maintainers:
+        - aescolar
+    files:
+        - boards/posix/nrf52_bsim/
+    labels:
+        - "platform: nrf52_bsim"
+
+"POSIX API layer":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - include/posix/
+        - lib/posix/
+        - tests/posix/
+    labels:
+        - "area: POSIX"
+
+"Power management":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - pizi-nordic
```

Comment:
Needs to be removed

Reply:
Ok, I understood for a slack discussion that pizi-nordic is not anymore working on Zephyr, so therefore he cannot be a maintainer. This would be just out of date information from the spreadsheet.
I will correct it.

Reply:
Correct, we need to update the spreadsheet as well.

Reply:
I will mark it here explicitly, so the TSC will be able to review and approve the maintainer change.